From 6d9f81ec15b7b18f8791981a0d65dc4f266022ca Mon Sep 17 00:00:00 2001
From: Brian Dolbec <dolsen@gentoo.org>
Date: Mon, 15 Jan 2018 16:27:54 -0800
Subject: [PATCH] pkg/buildbot_pkg.py: Remove development virtualenvironment
 hack

This hack is not needed/nor works reliably in non-virtualenv installs.
---
 pkg/buildbot_pkg.py | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/buildbot_pkg.py b/buildbot_pkg.py
index 89847b040..7788e12d6 100644
--- a/buildbot_pkg.py
+++ b/buildbot_pkg.py
@@ -33,17 +33,6 @@ import setuptools.command.build_py
 import setuptools.command.egg_info
 from setuptools import setup
 
-old_listdir = os.listdir
-
-
-def listdir(path):
-    # patch listdir to avoid looking into node_modules
-    l = old_listdir(path)
-    if "node_modules" in l:
-        l.remove("node_modules")
-    return l
-os.listdir = listdir
-
 
 def check_output(cmd):
     """Version of check_output which does not throw error"""
-- 
2.15.1

